packages/teleport/src/Users/UserTokenLink/UserTokenLink.story.test.tsx

```diff
@@ -2,6 +2,10 @@ import React from 'react';
 import * as story from './UserTokenLink.story';
 import { render } from 'design/utils/testing';
 
+jest
+  .spyOn(Date, 'now')
+  .mockImplementation(() => Date.parse('2021-04-08T07:00:00Z'));
```

Comment:
why this global mocks? (yes, jest clears up the mocks after each test by default but still there must be a good reason to do mock it this way)